views/components/info/control.es

```diff
@@ -170,6 +237,7 @@ const PoiControl = connect((state, props) => ({
   render() {
     return (
       <div className='poi-control-container'>
+        <script>{this.handleTouchbar(this.props.tbtriggered)}</script>
```

Comment:
This is used to call the function which will handle Touch bar input. As the state changes, the function handleTouchbar will be called with the arg of the button triggered